include/ghex/unstructured/communication_object_ipr.hpp

```diff
@@ -473,14 +474,13 @@ namespace gridtools {
 
                     template<typename Arch,
                              typename ValueType,
-                             typename BufferType,
                              typename Memory,
                              typename Halos,
                              typename Function,
                              typename DeviceIdType,
                              typename Pool,
                              typename Field = void>
-                    void allocate_send(Memory& memory,
+                    void allocate_send(Memory& memory, // reference not strictly needed here
```

Comment:
why is this comment useful? could you expand it?

Reply:
as before

Reply:
Same as for `set_recv_size`